console/war/src/main/java/nl/nn/adapterframework/runner/ConsoleFrontendBean.java

```diff
@@ -64,7 +71,23 @@ public void handleRequest(HttpServletRequest request, HttpServletResponse respon
 
 		};
 		SpringUtils.autowireByName(applicationContext, requestHandler);
-		requestHandler.setLocationValues(Arrays.asList("classpath:/META-INF/resources/iaf/gui/"));
+
+		String frontendFolder = "classpath:/META-INF/resources/iaf/gui/";
+		if(Arrays.asList(environment.getActiveProfiles()).contains("dev")){
+			String devFrontendLocation = environment.getProperty("frontend.resources.location");
+			if(devFrontendLocation == null) { // get default location based on current working directory
+				Path currentRelativePath = Paths.get("");
+				String basePath = currentRelativePath.toAbsolutePath().toString();
+				devFrontendLocation = basePath + "/console/frontend/target/frontend/";
```

Comment:
Het is mooier dit over te laten aan de Path of File api.

Reply:
Je bedoelt de Paths api het laten concatten en het dan pas naar string zetten?